### DISCUSSION TITLE
Cut out connection profiles

### DISCUSSION THREAD

**Discussion by NSASchweiz** [iterate-ch/cyberduck#15568] [Q&A]
Hello 

i dont want to give my users the opertunity, to chose the presetup profiles like alibaba, amazon etc.
How can i cut them out? or disable them in reg eg.

greetings

**Comment by dkocher** [comment]
You can disable the _Preferences → Profiles_ tab using the hidden configuration option in Cyberduck [^1] or Mountain Duck [^2].

```
preferences.profiles.enable=false
```

[^1]: https://docs.cyberduck.io/cyberduck/preferences/#hidden-configuration-options
[^2]: https://docs.cyberduck.io/mountainduck/preferences/#hidden-configuration-options

**Reply by NSASchweiz** [reply]
Hi dkocher
thx for the links and the instruction.

i now added the file C:\Users\[User]\AppData\Roaming\Cyberduck\default.properties
i added a new line in the file with: preferences.profiles.enable=false

after i started cyberduck, i still got the profiles :(

**Reply by ylangisc** [reply]
@dkocher On Windows this property is implemented for Mountain Duck only.

**Reply by NSASchweiz** [reply]
@ylangisc so if I understood correctly, there is actually no way to hide the preset profiles at the moment? At least for cyberduck.
